reporting/create_aggregate_FT_report.py

```diff
@@ -0,0 +1,118 @@
+import argparse
+import requests
+import sys
+import time
+from socotratools.client import SocotraClient
+from socotratools import dates
+import csv
+from itertools import groupby
+from operator import itemgetter
+
+# ETL application to aggregate financial transaction
+# report by lines of business.
+
+# Usage:
+# python create_aggregate_FT_report.py
+#       -n <hostname>
+#       -u <username>
+#       -n <password>
+#       -s <report start date>, e.g., 2018-7-01
+#       -e <report end date>, e.g., 2018-07-31
+#       -o <outputfile>, e.g., outfile
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description='Main Test Application')
+    parser.add_argument('-n', '--hostname', required=True)
+    parser.add_argument('-u', '--username',
+                        default='alice.lee', required=False)
+    parser.add_argument('-p', '--password', default='socotra', required=False)
+    parser.add_argument('-s', '--startdate', required=True)
+    parser.add_argument('-e', '--enddate', required=True)
+    parser.add_argument('-o', '--outputfile', required=True)
+
+    return parser.parse_args()
+
+
+def get_report_timestamps(startdate, enddate):
+    start_timestamp = dates.date_to_millis(startdate, 'Europe/Berlin', "%Y-%m-%d")
+    end_timestamp = dates.date_to_millis(enddate, 'Europe/Berlin', "%Y-%m-%d")
```

Comment:
Can we change these to "Americas/Los Angeles"?

Reply:
why?